aries_cloudagent/protocols/connections/v1_0/routes.py

```diff
@@ -373,16 +387,21 @@ async def connections_accept_invitation(request: web.BaseRequest):
     context = request.app["request_context"]
     outbound_handler = request.app["outbound_message_router"]
     connection_id = request.match_info["conn_id"]
+
     try:
         connection = await ConnectionRecord.retrieve_by_id(context, connection_id)
+        connection_mgr = ConnectionManager(context)
+        my_label = request.query.get("my_label") or None
+        my_endpoint = request.query.get("my_endpoint") or None
+        request = await connection_mgr.create_request(connection, my_label, my_endpoint)
+        result = connection.serialize()
     except StorageNotFoundError as err:
         raise web.HTTPNotFound(reason=err.roll_up) from err
-    connection_mgr = ConnectionManager(context)
-    my_label = request.query.get("my_label") or None
-    my_endpoint = request.query.get("my_endpoint") or None
-    request = await connection_mgr.create_request(connection, my_label, my_endpoint)
+    except (StorageError, WalletError, ConnectionManagerError, BaseModelError) as err:
```

Comment:
Should it just catch BaseError?

Reply:
I think it's OK to be specific here: catching an omnibus error will probably make LGTM-bot squawk.

Reply:
I'd prefer to see what can happen in particular.